photowall/app/elements/presentation-api/presentation-api.html

```diff
@@ -23,7 +23,7 @@
 Example:
 
     <presentation-api
-      url="http://google.com/cast/#__castAppId__=1624B172"
+      url="https://google.com"
```

Comment:
This seems like an odd example URL.  Maybe https://www.example.com/presentation.html

Reply:
Done